recipes/openblas/all/conanfile.py

```diff
@@ -50,6 +50,10 @@ def configure(self):
         if self.options.shared:
             del self.options.fPIC
 
+    def requirements(self):
```

Comment:
I believe gfortran is a build requirement, does not it? You want fortran to build lapack only.

Reply:
that is true. I should use build_requires I suppose there.